docs/faq.rst

```diff
@@ -24,18 +35,46 @@ Unfortunately, fMRIPrep's working directory structure is not stable across versi
 so writing code to grab the relevant files from the working directory is a bit of a moving target.
 Nevertheless, we have some code (thanks to Julio Peraza) that works for version 20.2.1.
 
-.. warning::
-    We will try to keep the following gist up-to-date, but there is no guarantee that it will work for a given version.
-    Use it with caution!
-
-    If you do find that the gist isn't working for an fMRIPrep version >= 20.2.1,
-    please comment on `Issue #717 <https://github.com/ME-ICA/tedana/issues/717>`_ (even if it's closed)
-    and we will take a look at the problem.
-
 .. raw:: html
 
     <script src="https://gist.github.com/tsalo/83828e0c1e9009f3cbd82caed888afba.js"></script>
 
+Warping scanner-space fMRIPrep outputs to standard space
+========================================================
+
+Here is a basic approach to normalizing scanner-space tedana-denoised data created from fMRIPrep outputs,
+using ANTS's antsApplyTransforms tool.
+The actual filenames of fMRIPrep derivatives depend on the filenames in the BIDS dataset
+(e.g., the name of the task, run numbers, etc.),
+but in this example we chose to use the simple example of "sub-01" and "task-rest".
+The standard space template in this example is "MNI152NLin2009cAsym", but will depend on fMRIPrep settings in practice.
+
+.. code-block:: bash
```

Comment:
Could someone take a look at this code? I've used it, but I don't know if it's the optimal set of settings.

Reply:
I don't know enough about either ants or fmriprep to make an informed choice about the settings, but if you ran it and it works as a starting point it's probably better to have something than nothing (especially since you note it will depend on the settings).

Reply:
This is silly, but can we put this in a gist and link it as we did above ? It would both standardize the formatting of the code snippets on this page and decouple updates from the docs.

Reply:
Also the code looks right from the last time I ran something like this ! I'll see if I can pull up a directory to actually run it on.

Reply:
I second the idea of a gist

Reply:
Done!